.github/workflows/build-and-push.yml

```diff
@@ -20,29 +21,64 @@ jobs:
       run: sudo ./.github/install-etcd.sh
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+
   build:
     needs: test
     runs-on: ubuntu-latest
+
     env:
-      IMAGE_NAME: kserve/modelmesh
+      IMAGE_NAME: modelmesh
+
     steps:
-    - uses: actions/checkout@v2
-    - name: Build and push runtime image
-      run: |
-        GIT_COMMIT=$(git rev-parse HEAD)
-        BUILD_ID=$(date '+%Y%m%d')-$(git rev-parse HEAD | cut -c -5)
+      - uses: actions/checkout@v2
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        if: github.event_name == 'push'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+      - name: export version variable
+        run: |
+          GIT_COMMIT=$(git rev-parse HEAD)
+          BUILD_ID=$(date '+%Y%m%d')-$(git rev-parse HEAD | cut -c -5)
+          
+          IMAGE_ID=kserve/$IMAGE_NAME
+
+          # Change all uppercase to lowercase
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
 
-        # Strip git ref prefix from version
-        VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+          # Strip "v" prefix from tag name
```

Comment:
should this be removed instead of being commented out? we do want the `v` prefix on the version tag

https://hub.docker.com/r/kserve/modelmesh/tags

Reply:
it's copied from the kserve workflow files https://github.com/kserve/kserve/blob/v0.10.0/.github/workflows/kserve-controller-docker-publish.yml#L71-L72

Reply:
fine to remove this as well

Reply:
lets remove it